src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -63,6 +67,14 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 	}
 
 	processEdit(edit: DesignerEdit): void {
+		const telemetryInfo = {
+			objectType: edit.path[0],
```

Comment:
I think i left a comment about this but i don't see it anymore. 
1. let's rename this to propertyName,
2. the path[0] is only the base property, please see the comments here to see how to come up with the correct object type: https://github.com/microsoft/sqltoolsservice/blob/main/src/Microsoft.SqlTools.ServiceLayer/TableDesigner/DesignerPathUtils.cs#L1 , if there are multiple properties in the path, let's use `prop1/prop2/prop3`